VAMobile/src/utils/hooks/index.tsx

```diff
@@ -32,17 +32,29 @@ import { useTheme as styledComponentsUseTheme } from 'styled-components'
  * @returns boolean to show the error
  */
 export const useError = (currentScreenID: ScreenIDTypes): boolean => {
-  const { errorsByScreenID } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  return useDowntime(DowntimeScreenIDToFeature[currentScreenID]) || !!errorsByScreenID[currentScreenID]
+  const { errorsByScreenID, downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
+  const features = ScreenIDToDowntimeFeatures[currentScreenID]
+  const downtime = oneOfFeaturesInDowntime(features, downtimeWindowsByFeature)
+  if (downtime) {
+    return true
+  }
+
+  return !!errorsByScreenID[currentScreenID]
 }
 
 export const useDowntime = (feature: DowntimeFeatureType): boolean => {
   const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  const mw = downtimeWindowsByFeature[feature]
-  if (!!mw && mw.startTime <= DateTime.now() && DateTime.now() <= mw.endTime) {
-    return true
-  }
-  return false
+  return featureInDowntime(feature, downtimeWindowsByFeature)
+}
+
+// this is not a hook. not sure this is the right place for this function
```

Comment:
These two are not actual hooks, but I didn't know where else to put them. Also, I found that I had to write them to receive `downtimeWindowsByFeature` rather than fetch them from state due to a linting issue saying you can't use hooks in callbacks. Very likely there's a better way of doing this.